### PR TITLE
Web3.js Woodpile

### PR DESCRIPTION
**Pull request by Woodpile37**
## Description

Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../blob/1.x/CONTRIBUTIONS.md).

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [x] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [x] I ran `npm run build` with success.
- [x] I have tested the built `dist/web3.min.js` in a browser.
- [x] I have tested my code on the live network.
- [x] I have checked the Deploy Preview and it looks correct.
- [x] I have updated the `CHANGELOG.md` file in the root folder.

### CODE REVIEW

File: packages/web3-eth-abi/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     },
     "main": "lib/index.js",
     "dependencies": {
-        "@ethersproject/abi": "5.0.7",
+        "@ethersproject/abi": "5.6.1",
```
Comment (jdevcs) [inline packages/web3-eth-abi/package.json]: @Woodpile37 There are merge conflicts could you fix those, and @ethersproject/abi package is updated in another PR https://github.com/ChainSafe/web3.js/blob/23e3045cdd54421ede10b3633d39e1a5576ee790/packages/web3-eth-abi/package.json#L17 so we can remove this changes of package and package lock in this PR.

**Review by jdevcs — CHANGES_REQUESTED** [review]

**Review by Woodpile37 — COMMENTED** [review]
Hey

**Comment by Woodpile37** [comment]
Thank you

File: packages/web3-eth-abi/package.json (hunk 1 above)
Comment (Woodpile37) [reply packages/web3-eth-abi/package.json]: 🤘

**Review by nazarhussain — CHANGES_REQUESTED** [review]
@Woodpile37 Please consider these points: 

1. Resolve the conflicts 
2. Remove the `package-lock.json` from your PR, as we will generate the file internally. 
3. Add entry to `CHANGELOG.md` to document what's been updated in this PR.

**Comment by nazarhussain** [comment]
@Woodpile37 Also update the title of the PR as well.

**Review by Woodpile37 — COMMENTED** [review]
Thank you

**Review by jdevcs — CHANGES_REQUESTED** [review]
@Woodpile37  ethersproject lib is already updated in another PR https://github.com/ChainSafe/web3.js/blob/35d8f7f76ca9412b714775cf15a2c1d57001020b/packages/web3-eth-abi/package.json#L17  , so could you remove changes in package and package lock files.

**Review by Woodpile37 — COMMENTED** [review]
Approve

**Review by avkos — CHANGES_REQUESTED** [review]
@Woodpile37 please remove package.json changes.
```"@ethersproject/abi": "5.6.2",```

**Review by jdevcs — COMMENTED** [review]
This PR is waiting for response from long time, so I am closing this PR. Feel free to come back and reopen.

**Pull request closed without merging**